modules/gitops-argo-cd-properties.adoc

```diff
@@ -59,26 +78,44 @@ a|* `<enabled>` - Toggle high availability support globally for Argo CD.
 
 |`initialRepositories`    |Initial Git repositories to configure Argo CD to use upon creation of the cluster.|`__<empty>__` |
 
+|`monitoring`    |Defines the workload status monitoring configuration status for your instance.| `__<object>__`
+a|* `<disableMetrics>` - Configure this field to enable or disable the collection of metrics for your instance.
+  * `<enabled>` - Indicates whether the workload status monitoring is enabled for your instance.
+
 |`notifications`    |Notifications controller configuration options.|`__<object>__`
 a|* `<enabled>` - The toggle to start the notifications-controller.
+  * `<env>` -  Environment to set for the notification controller workloads.
   * `<image>` - The container image for all Argo CD components. This overrides the `ARGOCD_IMAGE` environment variable.
   * `<version>` - The tag to use with the Notifications container image.
+  * `<replicas>` - The number of replicas to be executed for the Notifications controller.
   * `<resources>` - The container compute resources.
   * `<logLevel>` - The log level used by the Argo CD Application Controller component. Valid options are `debug`, `info`, `error`, and `warn`.
 
 |`repositoryCredentials`    |Git repository credential templates to configure Argo CD to use upon creation of the cluster.| `__<empty>__` |
 
-|`initialSSHKnownHosts`    |Initial SSH Known Hosts for Argo CD to use upon creation of the cluster.| `__<default_Argo_CD_Known_Hosts>__` |
+|`initialSSHKnownHosts`    |Defines Initial SSH Known Hosts data for Argo CD to use upon creation of the cluster for connecting Git repositories by SSH.| `__<default_Argo_CD_Known_Hosts>__`
+a|* `<Excludedefaulthosts>` - Indicates whether you want to add the default list of SSH Known Hosts provided by Argo CD.
+  * `<keys>` - Describes a custom set of SSH Known Hosts that you want to incorporate into your Argo CD server.
 
 |`kustomizeBuildOptions`    |The build options and parameters to use with `kustomize build`.|`__<empty>__` |
 
+|`kustomizeVersions`    |Defines a list of `Kustomize` versions that have been configured in the Argo CD Repo Server Container Image.|`__<empty>__`
+a|* `<path>` - The path of the `Kustomize` version in the file system of the Argo CD Repo Server container image.
+  * `<version>` - The `Kustomize` version in the `vX.Y.Z` format configured in the Argo CD Repo Server container image.
+
 |`oidcConfig` |The OIDC configuration as an alternative to Dex.|`__<empty>__` |
 
-|`nodePlacement` |Add the `nodeSelector` and the `tolerations`.|`__<empty>__` |
+|`nodePlacement` |Defines `NodeSelectors` and `Tolerations` for Argo CD workloads.|`__<empty>__`
+a|* `<NodeSelector>` - 	A map of key value pairs for node selection.
+  * `<Tolerations>` - Tolerations allow pods to create a schedule for nodes with matching taints.
 
 |`prometheus` |Prometheus configuration options.|`__<object>__`
 a|* `<enabled>` - Toggle Prometheus support globally for Argo CD.
```

Comment:
The line below this one (I cannot make a comment there:
from:
* `<host>` - The hostname to use for Ingress or Route resources.
to:
* `<host>` - The hostname to use for `Ingress` or `Route` resources.

Reply:
Fixed.